website/docs/gitops-templates/templates.mdx

```diff
@@ -0,0 +1,231 @@
+---
+title: Using templates
+sidebar_position: 3
+hide_title: true
+---
+
+import TierLabel from "../_components/TierLabel";
+
+# Using templates <TierLabel tiers="enterprise" />
+
```

Comment:
A little preamble here about what problems templates solve and a little example would be neat. @davidstauffer @JamWils do we have any material lying around we could include re: "why use templates"?

Reply:
Yes, we do! I will work on compiling the things we have from the initiative etc into a comprehensive why section. Should land before end of the week.